portfolio/src/main/java/com/google/sps/auth/User.java

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.auth;
+
+public final class User {
+  private final String id;
+  private final String name;
+  private final String logoutUrl;
```

Comment:
It is used on the AuthServlet and we are constructing the URL dynamically every time a user logged in a new URL is generated. Also the user should hold its own URL because it is unique for every user.